onecodex/viz/_bargraph.py

```diff
@@ -11,10 +11,11 @@
 )
 
 
-class VizBargraphMixin(object):
+class VizBargraphMixin:
     def plot_bargraph(
         self,
         rank=Rank.Auto,
+        metric=Metric.Auto,
```

Comment:
Do the plotting/stats methods need to have an `include_host` parameter?

Reply:
The plotting methods all use `include_host=False` which is consistent with the results pages. We could add this in the future (possibly via a new Metric).